include/unifex/await_transform.hpp

```diff
@@ -156,13 +156,17 @@ struct _awaitable_base<Promise, Value>::type {
   }
 };
 
+template <typename Promise, typename Sender>
+using _awaitable_base_t =
+  typename _awaitable_base<
+    Promise,
+    sender_single_value_return_type_t<remove_cvref_t<Sender>>>::type;
```

Comment:
Was the lack of `remove_cvref_t` the problem?

Reply:
Yes

Reply:
I wonder if the `sender_traits` metafunction should automatically strip reference qualifiers?